cardtel/utils/tests/test_game.py

```diff
@@ -0,0 +1,117 @@
+from django.test import TestCase
+
+from cardtel.models import (
+    Game,
+    Player,
+    Point,
+    User,
+)
+from cardtel.utils.gameplay import (
+    add_player_to_game,
+    create_game,
+    initialize_game,
+    increment_current_turn,
+    update_user_scores,
+    remove_player_from_game,
+)
+
+class TestGame(TestCase):
+    def setUp(self):
+        self.game = create_game()
+        self.user = User.objects.create(username='rmarincola')
+
+    def test_create_game(self):
+        game = create_game()
+        assert game.tables.count() == 1
+
+    def test_add_player_to_game(self):
+        game = self.game
+        user = self.user
+        player = add_player_to_game(game, user)
+        game = Game.objects.get(id=game.id)
+        assert game.players.count() == 1
+        assert player.play_order == 0
+
+        other_user = User.objects.create(username='bweems')
+        player = add_player_to_game(game, other_user)
+        game = Game.objects.get(id=game.id)
+        assert game.players.count() == 2
+        assert player.play_order == 1
+
+    def test_initialize_game(self):
+        game = self.game
+        user = self.user
+        player = add_player_to_game(game, user)
+        game = initialize_game(game)
+        assert game.current_turn == player
+
+        other_user = User.objects.create(username='bweems')
```

Comment:
I'd probably add this player to the game before the previous assertion to confirm that the order is actually being flipped by the score change